sources/us/wy/teton.json

```diff
@@ -29,6 +29,7 @@
             "st_type"
         ],
         "id": "pidn",
-        "accuracy": 2
+        "accuracy": 2,
+        "postcode": "msag_zip"
```

Comment:
Good catch!  Please correct the casing to `MSAG_zip` otherwise the machine won't pick it up.  You can also add `MSAG_city` and `state` for city and region, respectively.